README.md

```diff
@@ -1,12 +1,180 @@
-# im-github-deployments
+# IM GitHub Deployments Dashboard for Backstage
 
-Welcome to the im-github-deployments plugin!
+## What is the IM GitHub Deployments Dashboard
+The IM GitHub Deployments Dashboard plugin works in conjunction with the [im-open/create-github-deployment] GitHub workflow action to create and track GitHub [deployments] and [deployment statuses].
 
-_This plugin was created through the Backstage CLI_
 
 ## Getting started
 
-You can access the plugin by running `yarn start` in the root directory, and then navigating to [/im-github-deployments](http://localhost:3000/im-github-deployments).
-You can also serve the plugin in isolation by running `yarn start` in the plugin directory.
-This method of serving the plugin provides quicker iteration speed and a faster startup and hot reloads.
-It is only meant for local development, and the setup for it can be found inside the [/dev](./dev) directory.
+### Update `catalog-info.yaml`
+
+####  1. `github.com/project-slug` - REQUIRED
+
+The `catalog-info.yaml` file will need to be updated so that the `metadata.annotations` section contains a value for `github.com/project-slug`. The project slug is essentially concatenation of the GitHub owner and repository name separated by a forward slash, i.e., `im-open/im-github-deployments`.
+
+```yaml
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: im-github-deployemnts
+  ...
+  annotations:
+    github.com/project-slug: im-open/im-github-deployments
+```
+
+#### 2. `deployment-environments` - OPTIONAL, but recommended
+
+Adding the `metadata.deployment-environments` list allows the dashboard to be pre-populated with the expected deployment environments, and order their appearance in the dashboard.  Otherwise, the deployment environments will be ordered by how they were deployed and may not full reflect the project deployment lifecycle.
```

Comment:
```suggestion
Adding the `metadata.deployment-environments` list allows the dashboard to be pre-populated with the expected deployment environments, and order their appearance in the dashboard.  Otherwise, the deployment environments will be ordered by how they were deployed and may not fully reflect the project deployment lifecycle.
```